sdk/identity/azure-identity/azure/identity/_internal/shared_token_cache.py

```diff
@@ -243,4 +243,4 @@ def supported():
 
         :rtype: bool
         """
-        return platform.system() in {"Darwin", "Linux", "Windows"}
```

Comment:
Why do we need to remove Linux from the supported platforms? Shouldn't defaulting `allow_unencrypted` to `True` allow the `SharedTokenCacheCredential` to work on Linux as well?

Reply:
Thanks for being skeptical, trying to explain made me admit this change was a dubious solution to an unrelated problem, so I have reverted it.